README.md

```diff
@@ -1 +1,161 @@
-# audit-service-deploy
\ No newline at end of file
+## Audit Service Deployment
+
+### Introduction
+
+The Audit Service is designed to provide auditing of user and system actions by defining the required events and the information associated with each event.  This definition is registered with the Audit service and used to create a Java SDK to record the audited events.  The Audit service is multi-tenant aware.  Applications send events using the generated Java SDK to Elasticsearch where they are indexed according to each tenant.
+
+### Deployment Repository
+
+This repository provides the necessary files to easily get started using the Audit Service.
+
+The only pre-requisite required to get started is that Docker must be available on the system.
+
+The deployment files are in Docker Compose v3 format, and they are compatible with both Docker Compose and Docker Stack.
+
+The deployment file references at present only references Elasticsearch.
+
+### Demonstration
+
+The Docker Compose file contains the following services:
+
+
+<table><img src=./images/audit_service_deploy.jpg /></table>
+
+**Elasticsearch**  
+[Elasticsearch](https://www.elastic.co/products/elasticsearch) is a search engine based on [Lucene](https://lucene.apache.org/core/). It provides a distributed, multi-tenant capable full-text search engine with an HTTP web interface and schema-free JSON documents. Elasticsearch is developed in Java and is released as open source under the terms of the Apache License.
+
+### Usage
+
+1. Download the files from this repository  
+You can clone this repository using Git or else you can simply download the files as a Zip using the following link:  
+https://github.com/CAFAudit/audit-service-deploy/archive/develop.zip
+
+2. Configure the external parameters if required  
+The following parameters may be set:
```

Comment:
None of the parameters here look like external parameters.  They all look like internal parameters that are set in the docker-compose.yml file and cannot be set externally.